src/Service/ConnectionHandler.cpp

```diff
@@ -63,18 +63,26 @@ ConnectionHandler::ConnectionHandler(Context & global_context_, StreamSocket & s
     , keeper_dispatcher(global_context.getDispatcher())
     , operation_timeout(
           0,
-          Context::getConfigRef().getUInt("keeper.raft_settings.operation_timeout_ms", Coordination::DEFAULT_OPERATION_TIMEOUT_MS) * 1000)
+          Context::getConfigRef().getUInt(
+              "keeper.raft_settings.operation_timeout_ms", Coordination::DEFAULT_OPERATION_TIMEOUT_MS)
+              * 1000)
```

Comment:
I prefer the previous format. This code is too long.

Reply:
I change the column limit from 140 to 120.